pkgs/development/python-modules/cmigemo/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi,
+  six, pkgs
+}:
+
+buildPythonPackage rec {
+  pname = "cmigemo";
+  version = "0.1.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "7313aa3007f67600b066e04a4805e444563d151341deb330135b4dcdf6444626";
+  };
+
+  propagatedBuildInputs = [ six pkgs.cmigemo ];
+
+  preConfigure = ''
+    export LDFLAGS="-L${pkgs.cmigemo}/lib"
+    export CPPFLAGS="-I${pkgs.cmigemo}/include"
+    export LD_LIBRARY_PATH="${pkgs.cmigemo}/lib"
+  '';
+
+  postPatch = ''
+    sed -i 's~dict_path_base = "/usr/share/cmigemo"~dict_path_base = "/${pkgs.cmigemo}/share/migemo"~g' test/test_cmigemo.py
```

```suggestion
    sed -i 's~dict_path_base = "/usr/share/cmigemo"~dict_path_base = "/${cmigemo}/share/migemo"~g' test/test_cmigemo.py
```

Also can we use substituteInPlace here? The advantage is that it does not fail silently.